test/boot_requirements_checker_test.rb

```diff
@@ -40,7 +40,7 @@ def find_vol(mount_point, volumes)
     let(:storage_arch) { instance_double("::Storage::Arch") }
 
     before do
-      Yast::Storage::FakeProbing.new
+      Yast::Storage::StorageManager.fake_from_yaml
```

Comment:
calling fake_from_yaml(nil) seems an obvious semantic to me, but rubocop won't let me

my first thought was to call the method just 'fake'; maybe that is more understandable after all